{{cookiecutter.project_name}}/.pylintrc.toml

```diff
@@ -0,0 +1,52 @@
+[tool.pylint.main]
+ignore = [
+    ".git",
+    "__pycache__",
+    ".venv",
+    "venv",
+    "tests",
+    "migrations",
+    "settings",
+    "snapshots",
+]
+disable = [
+    "missing-module-docstring",
+    "missing-class-docstring",
+    "missing-function-docstring",
+    "unused-argument",
+    "redefined-builtin",
+    "import-outside-toplevel",
+    "no-else-return",
+    "unspecified-encoding",
+    "duplicate-code",
```

Comment:
Remove this from ignore? It is better to explicitly mark duplicated code for ignoring

Reply:
I agree, thank you.